exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/JoinUtils.java

```diff
@@ -127,6 +135,18 @@ public static boolean checkCartesianJoin(RelNode relNode, List<Integer> leftKeys
     return false;
   }
 
+  /**
+   * Overloaded version of {@link JoinUtils#checkCartesianJoin(RelNode, List, List, List)}
```

Comment:
Please change the Javadoc to reflect the aim of this method. (just copy it from the overloaded method)